From 69ff19adde4c70c739bf5a64a7cd7758fb25536d Mon Sep 17 00:00:00 2001
From: "Benjamin A. Beasley" <code@musicinmybrain.net>
Date: Fri, 19 Jan 2024 20:09:02 -0500
Subject: [PATCH 1/2] Add build-system.requires to pyproject.toml

This allows us to build the package with, e.g., python3 -m build
---
 pyproject.toml | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/pyproject.toml b/pyproject.toml
index 015024b..ca86fb0 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,3 +1,6 @@
+[build-system]
+requires = ["setuptools", "toml"]
+
 [tool]
 [tool.poetry]
 name = "sysrsync"

From cdedcfb11696f6af5c738658479a3b6ec3dbc7ca Mon Sep 17 00:00:00 2001
From: "Benjamin A. Beasley" <code@musicinmybrain.net>
Date: Fri, 19 Jan 2024 20:11:28 -0500
Subject: [PATCH 2/2] Fix typo in excluding tests from wheels

The wheels currently on PyPI look OK, but this keeps tests from being
inadvertently included when wheels are built by other means.
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 16f6655..f3ad689 100644
--- a/setup.py
+++ b/setup.py
@@ -15,7 +15,7 @@
     long_description=long_description,
     long_description_content_type="text/markdown",
     url="https://github.com/gchamon/sysrsync",
-    packages=setuptools.find_packages(exclude=['tests']),
+    packages=setuptools.find_packages(exclude=['test']),
     platforms='any',
     classifiers=[
         "Programming Language :: Python :: 3",
